hw/formal/fpv_vcf.tcl

```diff
@@ -0,0 +1,164 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script has commands to run the design using VC Formal
+
+#-------------------------------------------------------------------------
+# Analysing and elaborating the design
+#-------------------------------------------------------------------------
+# only one scr file exists in this folder
+
+analyze -format sverilog      \
+  -vcs "+define+FPV_ON -assert svaext \
+        +define+INC_ASSERT  \
```

Comment:
I think this is already defined in `ip/prim/rtl/prim_assert.sv`, is there any reason I missed that requires the define here again?

Reply:
Indeed, this should not be needed. See also:
https://github.com/lowRISC/opentitan/blob/517212025668e70f540edeffef11d651efbe5a6f/hw/formal/fpv.tcl#L25

Reply:
As we discusses, I think it is okay to leave this define here.